eZ/Bundle/EzPublishCoreBundle/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -0,0 +1,538 @@
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $iterationCount = (int) $input->getOption('iteration-count');
+        $this->dryRun = $input->getOption('dry-run');
+        $this->mode = $input->getOption('mode');
+
+        $from = $input->getOption('from');
+        $to = $input->getOption('to');
+
+        if ($from && !$this->validateDateTimeString($from, $output)) {
+            return;
+        }
+        if ($to && !$this->validateDateTimeString($to, $output)) {
+            return;
+        }
+
+        if ($from) {
+            $this->from = $this->dateStringToTimestamp($from);
+        }
+
+        if ($to) {
+            $this->to = $this->dateStringToTimestamp($to);
+        }
+
+        $consoleScript = $_SERVER['argv'][0];
+
+        if (getenv('INNER_CALL')) {
+            $this->timezone = $input->getArgument('timezone');
+            $this->processTimestamps((int) $input->getOption('offset'), $iterationCount);
+            $output->writeln($this->done);
+        } else {
+            $timezone = $input->getArgument('timezone');
+            $this->timezone = $this->validateTimezone($timezone, $output);
+
+            $modeTxt = 'Converting timestamps for both ezdate and ezdatetime fields.';
+
+            switch ($this->mode) {
+                case 'date':
+                    $modeTxt = 'Converting timestamps for ezdate fields.';
+                    break;
+                case 'datetime':
+                    $modeTxt = 'Converting timestamps for ezdatetime fields.';
+                    break;
+            }
+
+            $output->writeln(
+                [
+                    $modeTxt,
+                    'Calculating number of Content Object Attributes to update...',
+                ]
+            );
+            $count = $this->countEzDateObjectAttributes();
+            $output->writeln(
+                [
+                    sprintf('Found total of Content Objects Atributes for update: %d', $count),
+                    '',
+                ]
+            );
+
+            if ($count == 0) {
+                $output->writeln('Nothing to process, exiting.');
+
+                return;
+            }
+
+            $helper = $this->getHelper('question');
+            $question = new ConfirmationQuestion(
+                '<question>Are you sure you want to proceed?</question> ',
+                false
+            );
+
+            if (!$helper->ask($input, $output, $question)) {
+                $output->writeln('');
+
+                return;
+            }
+
+            $progressBar = $this->getProgressBar($count, $output);
+            $progressBar->start();
+
+            for ($offset = 0; $offset < $count; $offset += $iterationCount) {
+                $processBuilder = new ProcessBuilder(
```

Comment:
be aware this is deprecated in 3.4: https://api.symfony.com/3.4/Symfony/Component/Process/ProcessBuilder.html